cloudevents/spec.md

```diff
@@ -130,7 +130,15 @@ A "structured-mode message" is one where the event is fully encoded using
 a stand-alone event format and stored in the message body.
 
 A "binary-mode message" is one where the event data is stored in the message
-body, and event attributes are stored as part of message meta-data.
+body, and event attributes are stored as part of message metadata.
+
+Often, binary mode is used when the producer of the CloudEvent wishes to add
+the CloudEvents metadata to an existing event without impacting the message's
+body. This means that in most cases a binary CloudEvents enabled message would
```

Comment:
```suggestion
body. In most cases a CloudEvent encoded as a binary-mode message will
```
I guess it's not just typos - I can't help wordsmithing sometimes - take it or not.

Reply:
done!